whist/server/web_socket/subscriber.py

```diff
@@ -0,0 +1,27 @@
+"""Client abstraction"""
+from fastapi import WebSocket
+
+from whist.server.web_socket.events.event import Event
+
+
+# pylint: disable=too-few-public-methods
+class Subscriber:
+    """
+    A subscriber represents one client.
+    """
+
+    def __init__(self, connection: WebSocket):
+        """
+        Constructor
+        :param connection: Implementation of the web socket connection.
+        """
+        self._connection = connection
+
+    def send(self, event: Event) -> None:
+        """
+        Sends one event to this client.
+        :param event: Any type of event.
+        :return: None
+        """
+        wrapped_event = {event.name: event}
```

Comment:
this should probably be another format. I'd suggest:
```suggestion
        wrapped_event = {'name': event.name, 'event': event}
```